package.json

```diff
@@ -8,29 +8,38 @@
   "scripts": {
     "start": "flow-node app/index.js",
     "dev": "nodemon \"flow-node app/index.js\"",
+    "clean": "rm -rf app/circuits/* && rm -rf app/contracts/*",
+    "merkle": "flow-node app/utils/merkletree.js",
+    "contract": "flow-node app/utils/contract.js",
     "circuit:compile": "mkdir -p app/circuits && circom circuits/circuit.circom -o app/circuits/circuit.json",
-    "circuit:setup": "cd app/circuits && snarkjs setup",
-    "circuit:generateverifier": "mkdir -p contracts && cd app/circuits && snarkjs generateverifier && mv verifier.sol ../../contracts/Verifier.sol",
+    "circuit:setup": "cd app/circuits && snarkjs setup --protocol groth",
+    "circuit:generateverifier": "mkdir -p contracts && cd app/circuits && snarkjs generateverifier --protocol groth && mv verifier.sol ../../contracts/Verifier.sol",
     "circuit:test": "flow-node --experimental-worker app/utils/crypto.js"
   },
   "dependencies": {
     "body-parser": "^1.19.0",
     "circom": "0.0.31",
     "circomlib": "0.0.6",
```

Comment:
We should update this later when we get a chacne. This version has security bugs.